settings.py

```diff
@@ -106,21 +98,24 @@ def insert_debug_toolbar_middleware(middlewares):
         'client_id': env('FXA_CLIENT_ID', default='a25796da7bc73ffa'),
         'client_secret': env(
             'FXA_CLIENT_SECRET',
-            default='4828af02f60a12738a79c7121b06d42b481f112dce1831440902a8412d2770c5'),  # noqa
+            default='4828af02f60a12738a79c7121b06d42b481f112dce1831440902a8412d2770c5',
+        ),  # noqa
```

Comment:
I'm surprised these `# noqa` still work on the following line?

Reply:
Ah, I think those are no longer necessary ? black reformatted this for me... I'll remove them